tiled_test.go

```diff
@@ -252,13 +249,13 @@ func TestEmbeddedLoader(t *testing.T) {
 				if err != nil {
 					return nil, err
 				}
-				return loader.LoadFromReader("assets", file)
+				return LoadFromReader("assets", file)
```

Comment:
This should also provide assets FS, while currently not used this could be useful to improve test case that it correctly loads referenced resources from correct FS
```suggestion
				return LoadFromReader("assets", file, WithFileSystem(assetsFS))
```

Reply:
Don't understand the issue here. The file is opened from assets FS in this test:
```go
				file, err := assetsFS.Open("assets/test2.tmx")
```
You don't need to pass FS to `LoadFromReader` because it doesn't need the FS anymore - it reads from io.Reader

Reply:
It does read from reader but sequential files referenced in tmx file passed as reader could need access to FS (example to read template source for TMX Objects)

Reply:
I see. Fixed
Ideally we'd need to add a test which will fail if we try to load from non-embedded FS. Didn't figure out how to make this yet.